lib/WeBWorK3/Controller/ProblemSet.pm

```diff
@@ -31,6 +31,7 @@ sub getProblemSets {
 		$set->{set_visible} = $set->{set_visible} ? true : false;
 	}
 	$self->render(json => \@problem_sets);
+	return;
```

Comment:
Why the `return` statement here?  A return statement at the end of a sub that doesn't return anything is generally not needed.  I actually have a branch that removes all of the others in this file.  That is just clutter since the return values of these methods are not used for anything.

Reply:
This is something that comes up on perlcritic when I run it (but I forget what level).  Perhaps if we agree that we'll abide to perlcritic level 5 (I think that's the lowest), then I'll remove all of the unnecessary returns.

Reply:
Yeah, I thought that might be the case.  We can leave it I guess.

Reply:
Seems to be need for level 4 of perlcritic.  I guess we should just determine what level we want.